CONTRIBUTING.md

```diff
@@ -40,18 +40,31 @@ All code changes to the master branch must be done via [pull requests](https://h
 of the codebase named `[ticket_id]-[task_name]`.
 For example, branch name `19-ci-pipeline-setup` corresponds to [issue #19](https://github.com/Project-MONAI/MONAI/issues/19).
 Ideally, the new branch should be based on the latest `master` branch.
-1. Make changes to the branch ([use detailed commit messages if possible](https://chris.beams.io/posts/git-commit/)). If the changes introduce new features, make sure that you write [unit tests](#unit-testing).
+1. Make changes to the branch ([use detailed commit messages if possible](https://chris.beams.io/posts/git-commit/)).
```

Comment:
Why removed "If the changes introduce new features, make sure that you write [unit tests]", I feel it's necessary to highlight the unit tests.